common/game_rules/zz_hfe_vanilla_replace_game_rules.txt

```diff
@@ -217,6 +224,49 @@ can_build_branch_office_on_planet = {
 	}
 }
 
+
+# This = country
+can_be_part_of_galactic_community = {
+	OR = {
+		is_country_type = default
+		is_country_type = hfe_thoth
+		is_country_type = hfe_features_exodus
+		has_country_flag = galcom_form_aye
+		is_ai = no
+	}
+	NOR = {
+		has_civic = civic_fanatic_purifiers
+		has_civic = civic_hive_devouring_swarm
+		has_civic = civic_machine_terminator
+	}
+}
+
+# This = country
+can_see_galactic_community = {
+	always = yes
+#	OR = {
```

Comment:
Make no sense to break vanilla mechanic

Reply:
I made it so that Thoth can vote, remember. As for everyone seeing it, that should have been implemented in the first place. Seeing it does not mean you can do anything, and it doesn't affect the AI.